.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"editor.insertSpaces": false
+}
```

Comment:
I'm not sure this configuration file should be here. :thinking: 

It's configuration about your personal choice, and not project choice.
Moreover it's a development program specific configuration (VSCode)

Reply:
VSCode slips these in almost at easily as macOS does `.DS_Store` files. Might be worth adding to `.gitignore` when this file is removed.

Reply:
Sorry for the delayed response, @MacFJA. I've reverted the commit. :)

That said, I disagree with your middle sentence:

> It's configuration about your personal choice, and not project choice.

The choice to use tabs vs spaces should very much be a project choice since otherwise you run the risk of every change affecting every line because the whitespace keeps getting replaced every time people with opposite _personal_ preferences edit the same file.

That said, I can see the future that you want to avoid, where contributors end up spamming the project root with various editor configs. Speaking of which, might something like https://editorconfig.org/ be worth adopting?